data/102/555/091/102555091.geojson

```diff
@@ -186,7 +185,6 @@
             "continent_id":102191575,
             "country_id":85633041,
             "county_id":890456343,
-            "locality_id":101740755,
```

Comment:
So, in this case, the record currently has a dual hierarchy, but only one of the hierarchies is valid.. we should remove this entire hierarchy set so this feature only has one hierarchy (the one below, with "locality_id":101735893).

Reply:
How do I detect an valid/invalid hierarchy?

Reply:
I added https://github.com/whosonfirst-data/whosonfirst-data-admin-ca/pull/23/commits/a56eabe00c25c30f75ea9fe74162693841a4247b which I think is the correct fix.

> remove any hierarchies which are duplicates of, or a subset of another hierarchy